recipes-core/microcode/intel-microcode_%.bbappend

```diff
@@ -1,10 +1,19 @@
+# intel-microcode is MACHINE specific for some reason.
 
-# Install the microcode on the boot partition for Xen to load.
+# Since dom0 is shipped as a full image, the installer copies relevant files
+# from /boot to the boot partition.
 do_install_append() {
-	install -d ${D}/boot
-	install ${WORKDIR}/microcode_${PV}.bin ${D}/boot/microcode_${PV}.bin
-	cd ${D}/boot
-	ln -sf microcode_${PV}.bin microcode_intel.bin
+    install -d "${D}/boot"
+    # intel-microcode recipe installs in ${WORKDIR}.
+    install "${WORKDIR}/microcode_${PV}.bin" "${D}/boot/microcode_${PV}.bin"
+    ln -sfr "${D}/boot/microcode_${PV}.bin" "${D}/boot/microcode_intel.bin"
 }
 
-FILES_${PN} += " /boot"
+# Override do_deploy to suit OpenXT existing bootstrap.
+do_deploy() {
+    # intel-microcode recipe installs in ${WORKDIR}.
+    install "${WORKDIR}/microcode_${PV}.bin" "${DEPLOYDIR}/microcode_${PV}.bin"
+    ln -sfr "${DEPLOYDIR}/microcode_${PV}.bin" "${DEPLOYDIR}/microcode_intel.bin"
```

Comment:
```suggestion
    install "${WORKDIR}/microcode_${PV}.bin" "${DEPLOYDIR}/microcode_intel.bin"
```

Reply:
Don't we want the versioned file and the symlink?

Reply:
You are replacing this, so I think the version is unnecessary:
```
microcode_install() {
    cp -f ${IMAGE_ROOTFS}/boot/microcode_intel.bin ${DEPLOY_DIR_IMAGE}/microcode_intel.bin
}
```